components/webfield/ReviewerConsole.js

```diff
@@ -19,17 +19,16 @@ import ErrorDisplay from '../ErrorDisplay'
 import ReviewerConsoleMenuBar from './ReviewerConsoleMenuBar'
 import LoadingSpinner from '../LoadingSpinner'
 import ConsoleTaskList from './ConsoleTaskList'
+import { getProfileLink } from '../../lib/webfield-utils'
 
 const AreaChairInfo = ({ areaChairName, areaChairIds }) => (
   <div className="note-area-chairs">
-    <p>
-      <strong>{prettyField(areaChairName)}:</strong>{' '}
-      {areaChairIds.map((areaChairId) => (
-        <Link key={areaChairId} href={`/profile?id=${areaChairId}`}>
-          {prettyId(areaChairId)}{' '}
-        </Link>
-      ))}
-    </p>
+    <strong>{prettyField(areaChairName)}:</strong>
+    {areaChairIds.map((areaChairId) => (
+      <Link key={areaChairId} href={getProfileLink(areaChairId)}>
+        <div>{prettyId(areaChairId)}</div>
```

Comment:
Technically a `<div>` tag shouldn't be used inside an `<a>` tag... if we want to show a list of names with one name per line it would be better to use `<ul className="list-unstyled">`